samples/precompiled/Monitor.cs

```diff
@@ -135,11 +173,21 @@ internal class WeatherUnderground
             var callString = string.Format("http://api.wunderground.com/api/{0}/conditions/q/{1}/{2}.json", Environment.GetEnvironmentVariable("WeatherUndergroundApiKey"), location.State, location.City);
             var response = await httpClient.GetAsync(callString);
             var conditions = await response.Content.ReadAsAsync<JObject>();
+            var toString = conditions.ToString();
 
             JToken currentObservation;
             if (!conditions.TryGetValue("current_observation", out currentObservation))
             {
-                throw new ArgumentException("Could not find weather for this location. Try being more specific.");
+                var error = conditions.SelectToken("response.error"); //type, description
```

Comment:
nit: prefer using the explicit variable type instead of `var` in cases where the right-hand expression doesn't contain the type name, like here.